### DISCUSSION TITLE
v2.2.110

### DISCUSSION THREAD

**Discussion by Medard22** [Medard22/Dolphin-MMJR2-VBI#113] [Announcements]
Changelog for v2.2.110
- Fixed "Compile Shaders Before Starting" hang
- Small fix to FileUtil behavior

I had a report about a performance drop from v2.2.000. After extensive testing, I couldn´t reproduce it on my devices, nor was I able to track it to a specific code change. Every build has a slight minimal variance in performance. This can account for shader compilation, shader cache buildup, SoC thermal throttling, OS state, background apps, etc. When you first run any game on a new version of MMJR2, the shader cache is updated. There might be some minor dips as the shaders are rebuilt and cached that go away, after a while, or after you close and reload the game. There might be some performance hits on different SoCs, most likely due to architecture differences and GPU drivers, but I cannot test, nor fix that. If you are affected by a performance hit, you can go back to v2.2.000. Thanks to everyone who, helped me with testing. 

**Savestates from v2.2.000 and v2.2.100 are compatible.**  You should be able to update without any problems or revert back to older version if you have issues.

<hr /><em>This discussion was created from the release <a href='https://github.com/Medard22/Dolphin-MMJR2-VBI/releases/tag/v2.2.110'>v2.2.110</a>.</em>

**Comment by AkenoHononiku** [comment]
Why is there no audio?

**Reply by Medard22** [reply]
This is a first time I hear about audio issues. What is your hardware? What game are you trying to run?

**Comment by Leeghost9** [comment]
hi,

i have been using DMMjr2 for a while its one of the best emulator till now but there is a issue popping up in regular interval can you please fix it. (Android emulator)  
GFX FIFO: Unknown Opcode (0xef @ 0x7fd7f82797c0.preprocess=false).

**Reply by Medard22** [reply]
This usually means that emulated CPU and GPU desynced and it's mostly game specific. Try to disable Sync GPU on Skip Idle Hack in General settings and/or disable Dual Core. Both of these settings are hacks in a way. Some games needs tight CPU and GPU synchronization and there is really nothing I can do to fix it.

**Reply by Leeghost9** [reply]
Thanks

**Comment by Santi89rrjri** [comment]
There are trojans, don't download them! Don't download!

**Reply by Medard22** [reply]
If you download the releases from this Github repository then there are no trojans. Source code is available for review, feel free to build it yourself. Google play protect may fire a warning because my packages are signed with self-generated certificate and I am not a registered Android developer, but there is no malicious code involved.

**Comment by dan25064** [comment]
why doesn't the pointer work? i tried everything i could think of and it doesn't appear! pls help

**Reply by Medard22** [reply]
What game are you running? Sometimes speedhacks interferes with cursor function like in Super Mario Galaxy. Otherwise you can try go to Quick Settings -> Overlay Controls-> Motion Controls and click on "Don´t Use Device Sensors". The touchscreen pointer emulation should work then.

**Reply by dan25064** [reply]
thanks it works now!!!!! tho i have to change it from time to time. still, very thanks!!

**Comment by Dollyprune** [comment]
hello, I can't launch a mmjr2 game on my new device. It works on my mobile, my anburnic. I bought the kinhank pro 5x (Android box). When I launch a game it quits the direct program, I tried other versions it's the same.. no problem on dolphin or mmjr3. I uninstalled, cleared the cache nothing works thank you

**Reply by Dollyprune** [reply]
if I'm not mistaken, mmjr2 in version 1.9 had worked
